backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label :stock_location_id %>
```

Comment:
I'd just made it `<%= f.label Spree::StockLocation.model_name.human %> and it's working fine.  Everyone happy with that?  Or go for the more verbose option?